app/studio/[studioID]/view-locations/page.tsx

```diff
@@ -1,40 +1,61 @@
 'use client';
 
-// import Location from './location';
-// import React, { useEffect, useState } from 'react';
+import { redirect } from 'next/navigation';
+import { Session, User } from 'next-auth';
+import { useSession } from 'next-auth/react';
+import React, { useEffect, useState } from 'react';
 import AddLocation from '@/components/AddLocation';
+import Location from '../../../../src/components/Studios/location';
```

Comment:
use absolute imports for components. I'll make an absolute import for pages soon